src/longopt/grd2xyz_inc.h

```diff
@@ -25,26 +25,27 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'C', "",
-	          "",                  "",
+	{ 0, 'C', "rowcol|row_col",
 	          "",                  "",
+	          "f,i",               "one|fortran,indexz",
 		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
+	{ 0, 'L', "single",
```

Comment:
`hvline` means `horizontal` or `vertical` line of the matrix. Not perfect but I don't find `single` (what?) particularly better.